docs/quickstart_simple.md

```diff
@@ -58,7 +58,21 @@ npm install
 
 ## 4. Create a config file and update contract addresses
 
-Create a new config.js file:
+### Create a .env file
+
+```bash
+NETWORK_URL=http://172.15.0.3:8545
```

Comment:
what are these URLs?

Reply:
I believe these are urls from barge . `NETWORK_URL` is the rpc node url

Reply:
yes. It is the rpc node url